core-bundle/src/Controller/ContentElement/HyperlinkController.php

```diff
@@ -39,7 +39,7 @@ protected function getResponse(FragmentTemplate $template, ContentModel $model,
         $href = $this->insertTagParser->replaceInline($model->url);
 
         if (Validator::isRelativeUrl($href)) {
-            $href = $this->requestStack->getMainRequest()?->getBasePath().'/'.$href;
+            $href = $this->requestStack->getCurrentRequest()?->getBasePath().'/'.$href;
```

Comment:
Couldn’t we use `$request` here as this would be the current request?

Reply:
Of course. How could I miss this. 😄

Reply:
Changed in b7aa4467b64dc59776da5c0949b5bc8039fe70ee.